lib/builtin.pm

```diff
@@ -0,0 +1,93 @@
+package builtin;
+
+use strict;
+use warnings;
+
+# All code, including &import, is implemented by always-present functions in
+# the perl interpreter itself.
+# See also `builtin.c` in perl source
+
+1;
+__END__
+
+=head1 NAME
+
+builtin - Perl pragma to import built-in utility functions
+
+=head1 SYNOPSIS
+
+    use builtin qw( true false isbool );
+
+=head1 DESCRIPTION
+
+Perl provides several utility functions in a package called C<builtin::>.
+Functions in this namespace can be called by their fully-qualified names, but
+it is often desired to give them shorter name aliases for convenience.
+
+Individual named functions can be imported by listing them as import
+parameters on the C<use> statement for this pragma.
+
+=head2 Lexical Import
+
+This pragma module creates I<lexical> aliases in the currently-compiling scope
+to these builtin functions.
+
+    sub classify
+    {
+        my $sv = shift;
+
+        use builtin 'isbool';
+        return isbool($sv) ? "boolean" : "not a boolean";
+    }
+
+    # the isbool() function is no longer visible here
+    # but may still be called by builtin::isbool()
+
+Because these functions are imported lexically, rather than by package
+symbols, the user does not need to take any special measures to ensure they
+don't accidentally appear as object methods from a class.
+
+    package An::Object::Class {
+        use builtin 'true', 'false';
+        ...
+    }
+
+    # does not appear as a method
+    An::Object::Class->true;
+
+    # Can't locate object method "true" via package "An::Object::Class"
+    #   at ...
+
+=head1 FUNCTIONS
+
+=head2 true
+
+    $val = true;
+
+Returns the boolean truth value. While any defined but non-zero scalar value
+is considered "true" by perl, this one is special in that L</isbool> considers
+it to be a distinguished boolean value.
+
+=head2 false
+
+    $val = false;
+
+Returns the boolean fiction value. While non-true scalar value is considered
```

Comment:
`While non-true scalar value is considered` feels like a typo. Maybe `While any non-true scalar value is considered`

Reply:
That it is. Shall fix.